usr/sbin/p11sak/p11sak.c

```diff
@@ -844,13 +865,13 @@ static CK_RV tok_key_gen(CK_SESSION_HANDLE session, CK_ULONG keylength,
     CK_ATTRIBUTE tmplt[] = {
             // boolean attrs
             { CKA_TOKEN, &a_token, bs }, { CKA_PRIVATE, &a_private, bs }, {
-                    CKA_MODIFIABLE, &a_modifiable, bs }, { CKA_DERIVE,
-                    &a_derive, bs }, { CKA_SENSITIVE, &a_sensitive, bs }, {
-                    CKA_ENCRYPT, &a_encrypt, bs },
-            { CKA_DECRYPT, &a_decrypt, bs }, { CKA_SIGN, &a_sign, bs }, {
-                    CKA_VERIFY, &a_verify, bs }, { CKA_WRAP, &a_wrap, bs }, {
-                    CKA_UNWRAP, &a_unwrap, bs }, { CKA_EXTRACTABLE,
-                    &a_extractable, bs },
+            CKA_MODIFIABLE, &a_modifiable, bs }, { CKA_DERIVE, &a_derive, bs },
+            { CKA_SENSITIVE, &a_sensitive, bs }, {
+            CKA_ENCRYPT, &a_encrypt, bs }, { CKA_DECRYPT, &a_decrypt, bs }, {
+                    CKA_SIGN, &a_sign, bs }, {
+            CKA_VERIFY, &a_verify, bs }, { CKA_WRAP, &a_wrap, bs }, {
+            CKA_UNWRAP, &a_unwrap, bs },
+            { CKA_EXTRACTABLE, &a_extractable, bs },
```

Comment:
There a few chunks in this commit for which its hard to tell if you actually changed something. E.g., this seems to be formatting only. If you think the code needs re-formatting, please make it a separate commit next time.

Reply:
(The same applies to renaming variables or, more general, all changes that are unrelated to the commit message)

Reply:
Ok. Will do. Thanks for the suggestion.

Reply:
There is still a lot of formatting/renaming included in this commit. Thats okay for now, but please take care to put such changes in an extra commit (if its more than just a few lines) in future contributions.